src/background/events.ts

```diff
@@ -32,6 +32,10 @@ export const emitter = createNanoEvents<{
   chainsUpdated: () => void;
   chainChanged: (chain: Chain, origin: string) => void;
   'ui:chainSelected': (chain: Chain) => void;
+  globalError: (data: {
+    type: 'network_error' | 'signing_error';
```

Comment:
let's do `name` because it mirrors fields of the actual `Error` class

Reply:
ah, ok makes sense ✅